api/src/tm_clock.js

```diff
@@ -52,41 +52,49 @@ async function tmWorker () {
   // Get projects from TM2
   try {
     const db = conn()
-    const response = await TM.getProjects()
-    const json = JSON.parse(response)
+    let response = await TM.getProjects()
+    let json = JSON.parse(response)
     let features = ''
     switch (TM_VERSION) {
       case '3':
         features = json.results
-        if (features) {
-          // Map the features to objects for sql insertion
-          const sqlPromises = features.map(async (feature) => {
-            const rp = await TM.getProject(feature.projectId)
-            const geometry = JSON.parse(rp).areaOfInterest
-            const {
-              created: created_at, last_update: updated_at // eslint-disable-line camelcase
-            } = feature
-            return {
-              priority: feature.priority,
-              campaign_hashtag: feature.campaignTag,
-              created_at,
-              updated_at,
-              geometry,
-              name: feature.name,
-              description: feature.shortDescription,
-              validated: feature.percentValidated,
-              done: feature.percentMapped,
-              tm_id: feature.projectId
-            }
-          })
+        let sqlPromises = []
+        for (let page = 1; page <= json.pagination.pages; page++) {
```

Comment:
Using for loop with await means that we are making requests in sequence and not taking advantage of the async natures of js.

I suggest we use promise.all with along with a library like `p-limit` to control the concurrency. This should also considerably speed up the the process.